src/Tizen.Applications.ComponentManager/Tizen.Applications/ComponentInfo.cs

```diff
@@ -0,0 +1,259 @@
+            Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentInfoCreate(_componentId, out infoHandle);
+            if (err != Interop.ComponentManager.ErrorCode.None)
+            {
+                throw ComponentManager.ComponentManagerErrorFactory.GetException(err, "Failed to create the ComponentInfo.");
+            }
+            _infoHandle = infoHandle;
+        }
+
+        /// <summary>
+        /// Destructor of the class.
+        /// </summary>
+        ~ComponentInfo()
+        {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// Gets the component ID.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string ComponentId
+        {
+            get
+            {
+                if (!string.IsNullOrEmpty(_componentId))
+                    return _componentId;
+
+                string compId = string.Empty;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentInfoGetComponentId(_infoHandle, out compId);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the ComponentId. err = " + err);
+                }
+                _componentId = compId;
+
+                return _componentId;
+            }
+        }
+
+        /// <summary>
+        /// Gets the application ID of the component.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string ApplicationId
+        {
+            get
+            {
+                string appId = string.Empty;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentInfoGetAppId(_infoHandle, out appId);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the ApplicationId of " + _componentId + ". err = " + err);
+                }
+
+                return appId;
+            }
+        }
+
+        /// <summary>
+        /// Gets the type of the component.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public ComponentType Type
+        {
+            get
+            {
+                Interop.ComponentManager.ComponentInfoComponentType type = 0;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentInfoGetComponentType(_infoHandle, out type);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the Type of " + _componentId + ". err = " + err);
+                }
+
+                return (ComponentType)type;
+            }
+        }
+
+        /// <summary>
+        /// Checks whether the icon of the component should be displayed or not.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsIconDisplay
```

Comment:
```suggestion
        public bool IsIconDisplayed
```
or
```suggestion
        public bool IsIconShown
```

Reply:
Done. The name is changed to "IsIconDisplayed".